lib/kubeclient/common.rb

```diff
@@ -416,6 +428,18 @@ def api
       JSON.parse(response)
     end
 
+    def self.restclient_read_timeout_option
```

Comment:
can't make private at all, as it's a class method called from normal method.
I want the methods probing to get cached once, not per instance.
could make it a const but that feels even more public...